CONTRIBUTORS

```diff
@@ -9,3 +9,4 @@ Pantelis Elinas <pantelis.elinas@data61.csiro.au>
 Kevin Jung <kevin.jung@data61.csiro.au>
 Denis Khoshaba <denis.khoshaba@data61.csiro.au>
 Yuriy Tyshetskiy <yuriy.tyshetskiy@data61.csiro.au>
+Jeff Wang <Jeff.Wang@data61.csiro.au>
```

Comment:
I've already added you in `feature/gat` PR

Reply:
same order as gat one to avoid conflicts